blog/2021-07-15-supervisor_update.md

```diff
@@ -0,0 +1,80 @@
+---
+author: Joakim Sørensen
+authorURL: https://github.com/ludeeus
+authorImageURL: /img/profile/ludeeus.jpg
+authorTwitter: ludeeus
+title: Supervisor update
+---
+
+It's been a while since we posted about changes to the Supervisor, here is some highlights from the past year and the future.
+
+This information is mainly for add-on developers, but there are is little something for everyone in here.
+
+If you have not yet seen it, we have posted a [blog on the main site][main_blog] that you should read.
+
+## Snapshot -> Backup
+
+First up, as mentioned in the [blog on the main site][main_blog], we have started a transition away from the name "snapshot" that has been with us since the beginning of the Supervisor. And are now moving to the more recognizable "backup".
```

Comment:
```suggestion
First up, as mentioned in the [blog on the main site][main_blog], we have started a transition away from the name "snapshot" that has been with us since the beginning of the Supervisor and are now moving to the more recognizable "backup".
```